### DISCUSSION TITLE
Section 3: FundMe - Deploying to ZkSync

### DISCUSSION THREAD

**Discussion by Tala604** [Cyfrin/foundry-full-course-cu#2957] [Q&A] [answered]
Hi there!
I'm having trouble compiling `FundMe.sol.` The file is in the contracts folder and `default_workshop` directory, but it still won't compile. Could anyone help me figure out what’s going wrong?
![Screenshot (1053)](https://github.com/user-attachments/assets/c3ef0933-9c57-4919-8a11-6780e8a5f9e4)

**Comment by EngrPips** [comment, marked as answer]
Hello @Tala604, Please try solidity version `0.8.24` and let's see if that issue persist.

**Reply by Tala604** [reply]
Thank you! it works

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.
